sos/collector/sosnode.py

```diff
@@ -649,6 +647,15 @@ def finalize_sos_cmd(self):
             if self.opts.since:
                 sos_opts.append('--since=%s' % quote(self.opts.since))
 
+        # sos-4.0 changes the binary
+        if self.check_sos_version('4.0'):
+            self.sos_bin = 'sos report'
+
+        sos_cmd = sos_cmd.replace(
+            'sosreport',
```

Comment:
You relax from `if cmd.startswith('sosreport'):` test which is good ATM since `self.sos_cmd = 'sosreport --batch ..'`